.pre-commit-config.yaml

```diff
@@ -1,51 +1,56 @@
 exclude: Pipfile\.lock|migrations|\.idea
 
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.0.1
-    hooks:
-      - id: trailing-whitespace
-      - id: check-executables-have-shebangs
-      - id: check-json
-      - id: check-case-conflict
-      - id: check-toml
-      - id: check-merge-conflict
-      - id: check-xml
-      - id: check-yaml
-      - id: end-of-file-fixer
-      - id: check-symlinks
-      - id: mixed-line-ending
-      - id: fix-encoding-pragma
-        args:
-          - --remove
-      - id: pretty-format-json
-        args:
-          - --autofix
-      - id: requirements-txt-fixer
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.0.1
+  hooks:
+  - id: trailing-whitespace
+  - id: check-executables-have-shebangs
+  - id: check-json
+  - id: check-case-conflict
+  - id: check-toml
+  - id: check-merge-conflict
+  - id: check-xml
+  - id: check-yaml
```

Comment:
We can remove this check now I think 🙂

Reply:
`check-yaml` checks for syntactical validity. I suppose that's a good check to have?